sources/ci/build-and-test.md

```diff
@@ -13,6 +13,7 @@ The following sections of your **shippable.yml** will contain the bulk of your C
 -  The `post_ci` section includes commands that are not really a part of your core CI workflow but should be run after CI finishes. A good example is pushing artifacts to S3 or a Docker registry.
 -  The `on_success` includes commands you want to execute only if your CI workflow passes, i.e. the `ci` section exits with 0.
 -  The `on_failure` includes commands you want to execute only if your CI workflow fails, i.e. the `ci` section does not exit with a 0.
+-  The `always` includes commands you want to execute irrespective of CI workflow passes or fails.
```

Comment:
The `always` section includes commands that you want to execute whether the CI workflow passes or fails.